projects/frontend/data-pipelines/gui/cypress.json

```diff
@@ -23,7 +23,6 @@
   "env": {
     "hars_folders": "e2e/hars",
     "csp_url": "https://console-stg.cloud.vmware.com",
-    "login_url": "https://console-stg.cloud.vmware.com/csp/gateway/am/api/auth/api-tokens/authorize",
```

Comment:
Scripts are adapted to use base domain url for CSP `csp_url` with value `https://console-stg.cloud.vmware.com` and pathnames are hardcoded in the scripts, in that way Cypress env variables config is simplified.
Pathnames are very rarely changed, but domains are frequently changed because of different deployments and environments with different use-cases.
So with the new configuration we focus on domains and not if some pathname changes, which is breaking change and should be done with API versioning or backward compatibility which will work for us. Same way like Data Pipelines API.

Reply:
> Scripts are adapted to use base domain url for CSP csp_url with value https://console-stg.cloud.vmware.com and pathnames are hardcoded in the scripts

May you provide an example of such a hardcoded pathname in a script?

Reply:
Doesn't explain much. Could you link to such a hardcoded pathname in a script?

Reply:
it explain why.
this are places for CSP pathanames:
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-b1c629e18915230a7d69a22b06f9856f862eab39acfe5632aa5f408d28582a21R58
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-b1c629e18915230a7d69a22b06f9856f862eab39acfe5632aa5f408d28582a21R87

this are places for Data Pipelines APIs:
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR955https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR980
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR998
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR1016
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR1034
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR1052
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR1070
* https://github.com/vmware/versatile-data-kit/pull/1876/files#diff-d75b50e268731359be24cffb01032a5512486dcdb215575084f491b832e90c7fR1100
*

Reply:
I checked out the feature branch, and found
```
const REFRESH_TOKEN_START = 500;
const ORG_LINK_ROOT = '/csp/gateway/am/api/orgs/';
const CONSOLE_CLOUD_URL = 'https://console-stg.cloud.vmware.com/';
```
Is that what the links are pointing at (sorry -the links don't work on my side)
If so, then `/csp/gateway/am/api/orgs/`seems pretty CSP-specific to me. That's not an agile approach IMO, the entire URL needs to be configurable. May be in one place, so it's easier.

Reply:
* `projects/frontend/data-pipelines/gui/e2e/support/commands.js`
  * line 58
  * line 87
* `projects/frontend/data-pipelines/gui/e2e/plugins/helpers/job-helpers.plugins.js`
  * line 755
  * line 955
  * line 980
  * line 998
  * line 1016
  * line 1034
  * line 1052
  * line 1070
  * line 1100